<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ocularexam</class>
 <widget class="QDialog" name="ocularexam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="fixationPoint">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>210</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="instructionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>781</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Press Continue when the chin rest correctly configured.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="continueButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Continue</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>801</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="cointainerLayout"/>
  </widget>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>fixationPoint</zorder>
  <zorder>instructionLabel</zorder>
  <zorder>continueButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
